### DISCUSSION TITLE
[New Feature] Infantry Prone Toggle

### DISCUSSION THREAD

**Discussion by Onidotmoe** [Vinifera-Developers/Vinifera#1345] [Feature Requests]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I have attached as much information as possible *(screenshots, videos, links, etc)*.

### Description

Infantry go prone to reduce incoming damage but this can be very annoying if you're trying to get them to run away, someway to prevent them doing that would be nice.

Maybe inside a C&C3 style aggro menu in the bottom left corner of the screen?

**Comment by Rampastring** [comment]
You might already be aware but just a FYI, you can double-click when giving the units a move order to force them to stand up.

(I only learned this from browsing the TD/RA source code, just playing the games for 20+ years wasn't enough for finding this out :') )

**Comment by Onidotmoe** [comment]
Can i force them to go prone tho? 🤔
